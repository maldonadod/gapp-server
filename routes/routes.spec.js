const {
  routes
} = require('./index')

test('Map of routes should match snapshot', () => {
  
  expect(routes).toMatchSnapshot()
})