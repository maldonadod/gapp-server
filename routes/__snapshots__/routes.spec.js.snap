// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map of routes should match snapshot 1`] = `
Array [
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "get",
    "path": "/",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "post",
    "path": "/signup",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "post",
    "path": "/login",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "get",
    "path": "/places",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "get",
    "path": "/places",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "get",
    "path": "/profile/me",
  },
  Object {
    "handlers": Array [
      [Function],
      [Function],
    ],
    "method": "patch",
    "path": "/profile/me",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "get",
    "path": "/profile/me/events",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "get",
    "path": "/events/:_id",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "get",
    "path": "/events",
  },
  Object {
    "handlers": Array [
      [Function],
      [Function],
    ],
    "method": "post",
    "path": "/events",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "patch",
    "path": "/events/:_id",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "get",
    "path": "/contacts",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "get",
    "path": "/events/:chapter_id/guests",
  },
  Object {
    "handlers": Array [
      [Function],
    ],
    "method": "patch",
    "path": "/events/:chapter_id/guests/:guest_id",
  },
]
`;
